scripts/index.js

```diff
@@ -512,6 +512,20 @@
       });
     });
 
+    [
+      'llhls'
+    ].forEach(function(name) {
+      stateEls[name].checked = true;
+    });
+
+    [
+      'exact-manifest-timings',
+      'pixel-diff-selector',
+      'bufferBasedABR'
```

Comment:
This is throwing an error on [the Netlify preview build](https://deploy-preview-1301--videojs-http-streaming.netlify.app/):
```suggestion
      'buffer-water'
```